src/TestUtils/src/UITest.Appium/Actions/AppiumCatalystThemeChangeAction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UITest.Core;
+
+namespace UITest.Appium
+{
+	public class AppiumCatalystThemeChangeAction : ICommandExecutionGroup
+	{
+		const string SetLightTheme = "setLightTheme";
+		const string SetDarkTheme = "setDarkTheme";
+
+		readonly List<string> _commands = new()
+		{
+			SetLightTheme,
+			SetDarkTheme
+		};
+
+		public CommandResponse Execute(string commandName, IDictionary<string, object> parameters)
+		{
+			if (commandName == SetLightTheme)
+			{
+				ShellHelper.ExecuteShellCommand($"osascript -e 'tell app \"System Events\" to tell appearance preferences to set dark mode to false'");
```

Comment:
The Appium Mac driver not support it, so, use an AppleScript to change the settings.